src/admin/render_readmes.rs

```diff
@@ -160,7 +162,14 @@ fn get_readme(
         .uploader
         .crate_location(krate_name, &version.num.to_string());
 
-    let response = match client.get(&location).send() {
+    let location = match config.uploader {
+        Uploader::S3 { .. } => location,
+        Uploader::Local => format!("http://localhost:8888/{}", location),
+    };
```

Comment:
shouldn't that be taken care of inside the `crate_location()` method?

Reply:
I actually tried this approach at first, but it broke downloading readmes from the frontend. Frontend would get a redirect to the backend (:8888) and that caused a cors block, the branch with that version is [here](https://github.com/plaets/crates.io)